src/datasources/human-description-api/json/contract-descriptions.json

```diff
@@ -0,0 +1,14 @@
+{
+  "function transfer(address, uint256)": "Send {{tokenValue $1}} to {{address $0}}",
+  "function approve(address, uint256)": "Approve {{tokenValue $1}}",
+  "function withdraw(uint256)": "Withdraw {{tokenValue $0}}",
+  "function changeThreshold(uint256)": "Change threshold to {{decimals $0}}",
+  "function addOwnerWithThreshold(address,uint256)": "Add new owner {{address $0}} with threshold {{decimals $1}}",
+  "function removeOwner(address,address,uint256)": "Remove owner {{address $1}} with threshold {{decimals $2}}",
+  "function swapOwner(address,address,address)": "Swap owner {{address $1}} with {{address $2}}",
+  "function setGuard(address)": "Set guard {{address $0}}",
+  "function enableModule(address)": "Enable module {{address $0}}",
+  "function setFallbackHandler(address)": "Set fallback handler {{address $0}}",
+  "function setDelegate(bytes32, address)": "Set delegate {{address $1}}",
+  "function setAllowance(address,address,uint96,uint16,uint32)": "Set allowance for {{address $0}}"
```

Comment:
Why do we have a space between the arguments for `setDelegate` but not for `setAllowance`?

Reply:
Nice catch, I think it doesn't matter for getting the function signature so I will add a space to all of them for better readability.

Reply:
> Nice catch, I think it doesn't matter for getting the function signature so I will add a space to all of them for better readability.

The computed hash can potentially be different. Depends on the implementation of the library. I'll take a look.

Reply:
it seems that the `viem` trims the whitespace between the parameters so we should be good either way 👍 

https://github.com/wagmi-dev/viem/blob/169a695edeb7a0d1b6a9885509c3da2683cfe7ba/src/utils/contract/extractFunctionParts.ts#L17

I'd still keep one style to be consistent (preference for without spaces).